.Rhistory

```diff
@@ -0,0 +1,6 @@
+library("edgeR")
```

Comment:
@silviamorins What is this file doing? Its not there in the current `dev` code.

Reply:
You also have `edgeR` twice?

Reply:
Sorry, this was me messing up my repository.